### PR TITLE
#934: [typescript .d.ts] ResizeSensor (closes #934)

### LINKED ISSUE #934: [typescript .d.ts] ResizeSensor
← #908

## requirements
{describe the new feature}

## specs
{optional: describe technical specs to implement this feature, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by ascariandrea**
Closes #934

## Test Plan

> These results can be screenshots, query results, or even just pointers to unit tests that successfully passed.

- [ ] Google Chrome
- [ ] Internet Explorer

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.

### CODE REVIEW

File: src/resize-sensor/ResizeSensor.d.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { PureComponent } from 'react';
+import { Type } from 'tcomb';
+
+export type ResizeSensorProps = {
+  children: any,
+  onResize: (elements: HTMLElement) => void,
```
Comment (FrancescoCioria) [inline src/resize-sensor/ResizeSensor.d.ts]: I think `onResize` is called with no argument

Comment (ascariandrea) [inline src/resize-sensor/ResizeSensor.d.ts]: I digged a little into `css-element-queries` and found https://github.com/marcj/css-element-queries/blob/master/src/ResizeSensor.js#L52

Comment (FrancescoCioria) [inline src/resize-sensor/ResizeSensor.d.ts]: yeah, but your should look at `this.props.onResize` inside the component and right now it's called with no arg

Comment (ascariandrea) [inline src/resize-sensor/ResizeSensor.d.ts]: 👍 fixed!

**Pull request merged**